packages/truffle-box/tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "esModuleInterop": true,
+    "target": "es6",
+    "noImplicitAny": true,
+    "moduleResolution": "node",
+    "sourceMap": true,
+    "outDir": "dist",
+    "baseUrl": ".",
+    "lib": ["es2017"],
+    "paths": {
+      "web3": [
+        "../../node_modules/@types/web3/index",
+        "node_modules/web3/index"
+      ]
+    },
+    "rootDir": ".",
+    "typeRoots": [
+      "./typings",
+      "node_modules/@types",
```

Comment:
This might end up being brittle if some of those `@types` get moved to `../../node_modules`. It's probably best practice to follow the way `@types/mocha` does it below, and explicitly list all required type packages.

Reply:
Wouldn't that cause issues when using pkgs independently? 

Locally it appears `tsc` knows to recursively check node mod's for the types.